linux_os/guide/services/ntp/chronyd_sync_clock/tests/correct_value.pass.sh

```diff
@@ -2,7 +2,7 @@
 # packages = chrony
 
 {{% if product == 'ubuntu2204' -%}}
-echo "makestep 1 1" >> {{{ chrony_conf_path }}}
+echo "makestep 1 1" > {{{ chrony_conf_path }}}
```

Comment:
any reason for not being an append?

Reply:
According to the [latest doc](https://complianceascode.readthedocs.io/en/latest/templates/template_reference.html#key-value-pair-in-file),  `If multiple occurrences of key have conflicting values, the check will evaluate to fail.`